src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -468,9 +594,27 @@ void CodegenNeuronCppVisitor::print_make_instance() const {
     const auto codegen_float_variables_size = codegen_float_variables.size();
     for (int i = 0; i < codegen_float_variables_size; ++i) {
         const auto& float_var = codegen_float_variables[i];
-        printer->fmt_line("&_ml.template fpfield<{}>(0){}",
+        printer->fmt_line("&_ml.template fpfield<{0}>(0){1} /* {2} */",
+                          i,
+                          i < codegen_float_variables_size - 1 || codegen_int_variables.size() > 0
```

Comment:
`nitpick`: looking at this first time, it's not easy to think of what this 2nd parameter is and logic around it? a comment would be useful.